packages/xrpl/test/integration/utils.ts

```diff
@@ -1,45 +1,197 @@
 import { assert } from 'chai'
-import _ from 'lodash'
+import omit from 'lodash/omit'
+import throttle from 'lodash/throttle'
 import { decode } from 'ripple-binary-codec'
-import { Client, Wallet, AccountInfoRequest } from 'xrpl-local'
+import {
+  Client,
+  Wallet,
+  AccountInfoRequest,
+  type SubmitResponse,
+  TimeoutError,
+  NotConnectedError,
+  unixTimeToRippleTime,
+} from 'xrpl-local'
 import { Payment, Transaction } from 'xrpl-local/models/transactions'
 import { hashSignedTx } from 'xrpl-local/utils/hashes'
 
 const masterAccount = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh'
 const masterSecret = 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb'
 
-export async function ledgerAccept(client: Client): Promise<void> {
-  const request = { command: 'ledger_accept' }
-  await client.connection.request(request)
+async function sendLedgerAccept(client: Client): Promise<unknown> {
+  return client.connection.request({ command: 'ledger_accept' })
 }
 
-export function subscribeDone(client: Client, done: Mocha.Done): void {
+/**
+ * Throttles an async function in a way that can be awaited.
+ * By default throttle doesn't return a promise for async functions unless it's invoking them immediately.
+ * See CUR-4769 for details.
```

Comment:
What is CUR-4769? (Or where did this comment come from?)

Reply:
Removed